README.md

```diff
@@ -1,34 +1,34 @@
 # Feedback Notifier
 
-A simple API that sends feedbacks from a contact page on a website or any rest client to the configured slack.
+A simple API that sends feedbacks from a contact page on a website or any REST client to a configured Slack account.
 
 ## Installation
 
-To configure the feedback notifier, you have to create a database on mysql and configure the connection on `config.py` file.
+To configure the feedback notifier, you have to first create a database on MySQL and set the connection on the `config.py` file.
```

Comment:
i think "set the connection configurations" or somethink like this would be better :thinking: